cg/cli/workflow/nf_analysis.py

```diff
@@ -124,3 +128,22 @@ def report_deliver(context: CGConfig, case_id: str, dry_run: bool) -> None:
     except Exception as error:
         LOG.error(f"Could not create report file: {error}")
         raise click.Abort()
+
+
+@click.command("store-housekeeper")
+@ARGUMENT_CASE_ID
+@OPTION_DRY
+@click.pass_obj
+def store_housekeeper(context: CGConfig, case_id: str, dry_run: bool) -> None:
+    """Store a finished RNAFUSION and TAXPROFILER analysis in Housekeeper and StatusDB."""
+    analysis_api: NfAnalysisAPI = context.meta_apis[MetaApis.ANALYSIS_API]
+    housekeeper_api: HousekeeperAPI = context.housekeeper_api
+    status_db: Store = context.status_db
+
+    try:
+        analysis_api.store_analysis_housekeeper(case_id=case_id, dry_run=dry_run)
+    except Exception as error:
+        LOG.error(f"Could not store bundle in Housekeeper and StatusDB: {error}!")
+        housekeeper_api.rollback()
+        status_db.session.rollback()
```

Comment:
```suggestion
    except StoreHousekeeperError as error:
        LOG.error(f"Could not store bundle in Housekeeper and StatusDB: {error}!")
```

Reply:
Perhaps we can add another `Exception` handler for some error that we have not foreseen. If that is needed